### DISCUSSION TITLE
.netstandard2.0 compatibility

### DISCUSSION THREAD

**Discussion by kishoreallwynraj** [aws/aws-ssm-data-protection-provider-for-aspnet#63] [Q&A] [answered]
Hello.. I noticed the target framework setup in this project is not having ".netstandard2.0". Is there any specific this nuget is not supporting ".netstandard2.0".?. 
I tried to recompile the project to target ".netstandard2.0" and updated my .NET6 project to use this recompiled project in ".netstandard2.0" and still it was working only.
So trying to understand if there is any known compatibility issues there in ".netstandard2.0" which i am not able to foresee now.. please clarify.

**Comment by ashishdhingra** [comment, marked as answer]
@kishoreallwynraj Please refer to similar requirement for different high-level library https://github.com/aws/aws-dotnet-extensions-configuration/pull/150/files. There is a comment from @normj on why the other library was missing `NetStandard` support. Do you have a requirement to use this library .NET Framework projects?

CCing @normj for any additional inputs.
